### DISCUSSION TITLE
Vertical scroll in swiper-slides

### DISCUSSION THREAD

**Discussion by jc9292** [nolimits4web/swiper#6780] [Q&A]
Hi,

Is there a swiper.js demo with horizontal swipe between slides AND vertical scroll in each slide ? (see [image](https://i.imgur.com/ov0mrQs.pngl))

I've try to set "overflow-y:scroll", but that doesn't work properly.

Thanks.

**Comment by DesmonFur** [comment]
I was looking for the same, this could work for your use purpose: https://alvarotrigo.com/fullPage/#page1

I'm currently working on making it work in Swiper for fun.
